ci-operator/config/openshift/ols-load-generator/OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
+- bparees
+- xrajesh
+- eparis
+options: {}
+reviewers:
+- bparees
+- xrajesh
+- eparis
```

Comment:
Do we need eparis as reviewer and approver - I dont think so .  Could we add @tisnik as approver and reviewer. 
Can add @raptorsun as a reviewer.

Reply:
Updated the list as suggested. Keeping `eparis` only as an approver, if that sounds okay.